auth/README.md

```diff
@@ -103,6 +103,6 @@ If `MF_EMAIL_TEMPLATE` doesn't point to any file service will function but passw
 ## Usage
 
 For more information about service capabilities and its usage, please check out
-the [API documentation](openapi.yml).
+the [API documentation](https://api.mainflux.io/?urls.primaryName=auth-openapi.yml).
```

Comment:
Why this url?

Reply:
Yes @manuio , raw yml file is there. But on api.mainflux.io domain, it will be shown in SwaggerUI which is more user friendly